cmd/security-profiles-operator/main.go

```diff
@@ -540,7 +541,15 @@ func runWebhook(ctx *cli.Context, info *version.Info) error {
 
 	port := ctx.Int("port")
 
-	webhookServer := webhook.NewServer(webhook.Options{Port: port})
+	disableHTTP2 := func(c *tls.Config) {
+		c.NextProtos = []string{"http/1.1"}
+	}
+	webhookServerOptions := webhook.Options {
+		Port:    port,
+		TLSOpts: []func(config *tls.Config){disableHTTP2},
+	}
```

Comment:
Can we fix the verify CI here:
```
cmd/security-profiles-operator/main.go:547: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(sigs.k8s.io/security-profiles-operator) (gci)
```

Reply:
I'm also adding a testCase to ensure the endpoint is using HTTP/1.1.
Based on https://github.com/ComplianceAsCode/compliance-operator/pull/451